packages/firebase_messaging/firebase_messaging/pubspec.yaml

```
@@ -5,28 +5,28 @@ homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/f
 version: 8.0.0-dev.14
 
 environment:
-  sdk: ">=2.12.0-0 <3.0.0"
+  sdk: '>=2.12.0-133.7.beta <3.0.0'
   flutter: ">=1.12.13+hotfix.5 <2.0.0"
 
 dependencies:
-  meta: ^1.0.4
-  flutter:
-    sdk: flutter
-  firebase_core: ">=0.8.0-1.0.nullsafety.0 <0.8.0-2.0.nullsafety.0"
-  firebase_core_platform_interface: ">=4.0.0-1.0.nullsafety.0 <4.0.0-2.0.nullsafety.0"
+  firebase_core: ^0.7.0
+  firebase_core_platform_interface: ^3.0.1
```

```suggestion
  firebase_core: ">=0.8.0-1.0.nullsafety.0 <0.8.0-2.0.nullsafety.0"
  firebase_core_platform_interface: ">=4.0.0-1.0.nullsafety.0 <4.0.0-2.0.nullsafety.0"
```

Should be using nullsafety versions.